velox/dwio/common/SelectiveStructColumnReader.cpp

```diff
@@ -61,13 +61,12 @@ void SelectiveStructColumnReaderBase::next(
       numValues -= bits::countBits(mutation->deletedRows, 0, numValues);
     }
 
+    auto resultRowVector = std::dynamic_pointer_cast<RowVector>(result);
     // no readers
     // This can be either count(*) query or a query that select only
     // constant columns (partition keys or columns missing from an old file
     // due to schema evolution)
-    result->resize(numValues);
-
-    auto resultRowVector = std::dynamic_pointer_cast<RowVector>(result);
+    resultRowVector->setSize(numValues);
```

Comment:
No need to change it because the row vector will not have any non-constant child in this case